op-chain-ops/genesis/layer_two_test.go

```diff
@@ -34,7 +33,7 @@ func init() {
 var testKey, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
 
 // Tests the BuildL2MainnetGenesis factory with the provided config.
```

Comment:
Its good to leave this test for now but we should def move away from it in the future and rely completely on the solidity test coverage which i do believe almost already covers this besides the "use gov token" flag